Error when building in RPi4

<!--
Before submitting this issue, please make sure you have:
 
If you believe your issue to be a bug, please make sure you check the wiki page: https://github.com/MultiMC/MultiMC5/wiki/Report-a-Bug
-->

System Information
-----------------------------
MultiMC version:latest git clone from 06/February/2021

Operating System: Linux Mate 64bit running on Raspberry Pi 4

Summary of the issue or suggestion:
----------------------------------------------
When building, the cmake configures the build with no errors but as I execute "make -j4"  I get the following eror:

[ 42%] Building CXX object api/logic/CMakeFiles/MultiMC_logic.dir/minecraft/Library.cpp.o
[ 42%] Building CXX object api/logic/CMakeFiles/MultiMC_logic.dir/minecraft/VersionFile.cpp.o
[ 42%] Building CXX object api/logic/CMakeFiles/MultiMC_logic.dir/minecraft/VersionFilterData.cpp.o
[ 42%] Building CXX object api/logic/CMakeFiles/MultiMC_logic.dir/minecraft/World.cpp.o
[ 42%] Building CXX object api/logic/CMakeFiles/MultiMC_logic.dir/minecraft/WorldList.cpp.o
[ 43%] Building CXX object api/logic/CMakeFiles/MultiMC_logic.dir/minecraft/mod/Mod.cpp.o
/home/gian/MultiMC/src/api/logic/minecraft/World.cpp: In member function ‘void World::loadFromLevelDat(QByteArray)’:
/home/gian/MultiMC/src/api/logic/minecraft/World.cpp:474:17: error: catching polymorphic type ‘class std::out_of_range’ by value [-Werror=catch-value=]
  474 |     catch (std::out_of_range) {}
      |                 ^~~~~~~~~~~~
[ 43%] Building CXX object api/logic/CMakeFiles/MultiMC_logic.dir/minecraft/mod/ModFolderModel.cpp.o
[ 43%] Building CXX object api/logic/CMakeFiles/MultiMC_logic.dir/minecraft/mod/ModFolderLoadTask.cpp.o
[ 43%] Building CXX object api/logic/CMakeFiles/MultiMC_logic.dir/minecraft/mod/LocalModParseTask.cpp.o
cc1plus: all warnings being treated as errors
make[2]: *** [api/logic/CMakeFiles/MultiMC_logic.dir/build.make:1116: api/logic/CMakeFiles/MultiMC_logic.dir/minecraft/World.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[2]: Leaving directory '/home/gian/MultiMC/build'
make[1]: *** [CMakeFiles/Makefile2:1313: api/logic/CMakeFiles/MultiMC_logic.dir/all] Error 2
make[1]: Leaving directory '/home/gian/MultiMC/build'
make: *** [Makefile:141: all] Error 2
gian@RPi4:~/MultiMC/build$ 



What should happen:
------------------------------

build with no errors

Steps to reproduce the issue (Add more if needed):
-------------------------------------------------------------
1. clone the repository  
2. create build and install directories
3. cd build
3. cmake -DCMAKE_INSTALL_PREFIX=../install ../src
4. make -j4

Suspected cause:
---------------------------
No idea.

Logs/Screenshots:
----------------------------
[//]: # (Please refer to https://github.com/MultiMC/MultiMC5/wiki/Log-Upload for instructions on how to attach your logs.)


Additional Info:
---------------------------

--- a/api/logic/minecraft/World.cpp
+++ b/api/logic/minecraft/World.cpp
@@ -471,7 +471,7 @@ void World::loadFromLevelDat(QByteArray data)
         auto &WorldGen_val = val.at("WorldGenSettings");
         randomSeed = read_long(WorldGen_val, "seed");
     }
-    catch (std::out_of_range) {}
+    catch (const std::out_of_range &) {}
     if(!randomSeed) {
         randomSeed = read_long(val, "RandomSeed");
     }

